scripts/test

```diff
@@ -20,6 +20,7 @@ then
     then
         usage
     else
+        echo "Running Taui tests..."
         docker-compose run --rm taui test
     fi
```

Comment:
I originally added `./scripts/manage test` here but it's failing without collecting static files beforehand. We're [transitioning away from Travis CI](https://github.com/azavea/echo-locator/pull/438/files) so I'm taking it off for now until that's complete and then adding any necessary pre-test commands to the future `scripts/cibuild` file.